rhizome/host/bin/prep_host.rb

```diff
@@ -89,6 +89,7 @@
 # debugging crashes.
 r "apt-get -y install nvme-cli systemd-coredump" if is_prod_env
 
+r "apt-get update && apt-get -y install smartmontools"
```

Comment:
should this be guarded by is_prod_env as well? I'm looking at `nvme-cli` and thinking, "but isn't that pretty similar?"